src/output.cpp

```diff
@@ -354,6 +354,18 @@ void center_print(WINDOW *w, int y, nc_color FG, const char *mes, ...)
     mvwprintz(w, y, x, FG, "%s", text.c_str());
 }
 
+void right_print( WINDOW *w, const int line, const int available_width, const nc_color FG,
```

Comment:
I'd replace `available_width` with a `right_indednt`. It seems more consistent with the "left_print" functions (where `x` is essenitally a left indent) and doesn't enforce both knowing the actual width of the window and calculating a new one. Therefore would be shorter and more expressive in most cases.